utils/index.d.ts

```diff
@@ -1563,7 +1563,10 @@ export declare namespace AzExtFsExtra {
     export function ensureDir(resource: Uri | string): Promise<void>;
     export function ensureFile(resource: Uri | string): Promise<void>;
     export function readFile(resource: Uri | string): Promise<string>;
-    export function writeFile(resource: Uri | string, contents: string): Promise<void>;
+    /**
+     * @param seperator By default, will append "\r\n\r\n" between existing content and new content to be appended
+     */
+    export function appendFile(resource: Uri | string, contents: string, seperator?: string): Promise<void>;
```

Comment:
Nit:
```suggestion
    export function appendFile(resource: Uri | string, contents: string, separator?: string): Promise<void>;
```

Applies in the comment and the method definition as well